Public/Sdk/Public/Managed/runtimeConfigFiles.dsc

```diff
@@ -211,11 +211,11 @@ namespace RuntimeConfigFiles {
             }
         } : {};
 
-        // when not using Server GC, in large builds the front end is likely to get completely bogged
+        // when not using Server GC, in large builds the front end is likely to get completely bogged, on Unix we are currently disabling this

```

Comment:
It's clear from below that on non-"win" we are disabling this; instead, say here why